numpy/testing/_private/parameterized.py

```diff
@@ -179,7 +155,7 @@ def parameterized_argument_value_pairs(func, p):
 
     named_args = argspec.args[arg_offset:]
 
-    result = lzip(named_args, p.args)
+    result = list(zip(named_args, p.args)
```

Comment:
```suggestion
    result = list(zip(named_args, p.args))
```

Reply:
Still applies

Reply:
Do you know why this seems to trigger only on a small subset of the test environments?

Reply:
Only azure, travis, and shippable actually run the test suit, the other CI checks documentation, etc. Maybe you mean that?

Reply:
This doesn't fail locally when I test with `runtest.py -m full`
And I remember it as fail only in some of the azure pipelines.

Reply:
Dunno, best guess: pytest used to ignore errors during collecting tests (no reference, just based on a hunsh that there was a time when I did not have to skip such errors – when running tests in valgrind). So if the pytest version is off, it might only print out something scary at the beginning, but never notice this.